test/TestConstants.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <string>
+
+namespace TestConstants {
+    const std::string TEST_DIR(boost::filesystem::path(__FILE__).parent_path().string());
```

Comment:
The vs code treats the current_directory based on the cwd of the vs code config, instead of the actual test executable. So in our case the cwd from the terminal shows something like 
`/home/tigran/Documents/_current/QuickTurns/build/test/`
but from within the vs code it shows
`/home/tigran/Documents/_current/QuickTurns/`.
The only time the program gives a consistent behavior is with the __FILE__ macro.

Reply:
You're right. I refactored this.